File: Assignment-40/Program5.java
import  java.util.*;

class Program5
{
      public static void main(String arg[])   
      {
            Pattern pobj = new Pattern();
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter numbers of rows :");
            int i = sobj.nextInt();

            System.out.println("Enter numbers of colum :");
            int j = sobj.nextInt();

            pobj.Display(i,j);
      }
}
class Pattern
{
      public void Display(int iRow ,int iCol)
      {
            for(int i = 1;i<=iRow;i++)
            {
                  for(int j = 1;j<=iCol;j++)
                  {
                        if((j==1)||(j==iCol))
                        {
                              System.out.print(j+"\t");
                        }
                        else if((j>=i)||(i==iRow))
                        {
                              System.out.print(j+"\t");
                        }
                        else
                        {
                              System.out.print(" \t");
                        }
                  }
                  System.out.println();
            }
      }
}

File: Assignment-20/program5.c
#include<stdio.h>

void Display(int iRow ,int iCol)
{
      int i = 0 ,j = 0;
      int No = 1;

      for(i = 1;i <= iRow;i++)
      {
            
            for(j = 0;j < iCol;j++)
            {
                 if(j < No) 
                 {
                   printf("%d\t",No);
                   No++;
                 }
                      
                  
            }

            printf("\n");
      }
}

int main()
{
      int ivalue1 = 0;
      int ivalue2 = 0;

      printf("Enter The Nunmber Row :");
      scanf("%d",&ivalue1);

      printf("Enter The Number of Colum :");
      scanf("%d",&ivalue2);

      Display(ivalue1,ivalue2);

      return 0;
}